platform/ui/src/components/ViewportActionBar/ViewportActionBar.jsx

```diff
@@ -245,6 +249,8 @@ const ViewportActionBar = ({
     ? '#020424'
     : null;
 
+  const textColor = isPrimary ? 'text-study-primary' : 'text-study-secondary';
```

Comment:
can we keep the study date as white for current actually? 

for example, `const textColor = isPrimary ? '' : 'text-study-secondary';` would work